eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryTest.java

```diff
@@ -118,6 +118,14 @@ public void testCountByIdsAndComplexFilters(){
     }
 
     @Test
+    public void testCountByIdsAndComplexFiltersZeroCount(){
```

Comment:
Run the auto-formatting on this class, as the opening brackets should be preceded by a whitespace.